modules/custom/moj_resources/src/Utilities.php

```diff
@@ -93,7 +103,7 @@ public static function getPrisonsFor($node) {
     $prisons = array();
 
     foreach ($node->field_moj_prisons as $prison) {
-      array_push($prisons, $prison->target_id);
+      array_push($prisons, intval($prison->target_id));
```

Comment:
After finding some of these are coming through as strings when doing the Term API I don't trust these.

Reply:
So this does casting similar to `(int) $prisonCategory->target_id`?

Reply:
Yep, it's basically the same as `Number.parseInt()` in JS.